tests/hash-backend-performance/rollup.config.js

```diff
@@ -0,0 +1,40 @@
+import commonjs from "@rollup/plugin-commonjs";
+import nodeResolve from "@rollup/plugin-node-resolve";
+import typescript from "@rollup/plugin-typescript";
+
+const production = !process.env.ROLLUP_WATCH;
+
+const outdir = (fmt, env) => {
+  if (env === "node") {
+    return `dist/node`;
+  } else {
+    return `dist/${fmt}${env === "slim" ? "-slim" : ""}`;
+  }
+};
+
+const rolls = (fmt, env) => ({
+  input: "src/main.ts",
+  output: {
+    dir: outdir(fmt, env),
+    entryFileNames: `[name].${fmt === "cjs" ? "cjs" : "js"}`,
+    format: fmt,
+    name: "hash-backend-performance",
+    sourcemap: !production,
+  },
+  plugins: [
+    commonjs(),
+    typescript({
+      declaration: true,
+      declarationDir: outdir(fmt, env),
+      outDir: outdir(fmt, env),
+      rootDir: "src",
+      sourceMap: !production,
+      inlineSources: !production,
+      outputToFilesystem: false,
+    }),
+    nodeResolve(),
+  ],
+  external: ["@ory/client"],
+});
+
+export default [rolls("cjs", "fat")];
```

Comment:
Unless we need to ever change this config, I'd simplify it to get rid of the variables we're passing around that are constant values. It makes it easier to understand later, because we don't have to figure out what we're using the `env` and `fmt` variables to change and why (nothing)

```js
const outDir = "dist/cjs";

export default [{
  input: "src/main.ts",
  output: {
    dir: outDir,
    entryFileNames: "[name].cjs",
    format: "cjs",
    name: "hash-backend-performance",
    sourcemap: !production,
  },
  plugins: [
    commonjs(),
    typescript({
      declaration: true,
      declarationDir: outDir,
      outDir,
      rootDir: "src",
      sourceMap: !production,
      inlineSources: !production,
      outputToFilesystem: false,
    }),
    nodeResolve(),
  ],
  external: ["@ory/client"],
}];
```

Reply:
I simplified the rollup config and made it a TS file instead